File: PRAK605_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.c
#include <stdio.h>

int main() {
    int BarisKolom;
    printf("Masukkan jumlah baris/kolom: ");
    scanf("%d", &BarisKolom);

    printf("Matriks A\n");
    int matrixA[BarisKolom][BarisKolom];
    for (int i = 0; i < BarisKolom; i++) {
        for (int j = 0; j < BarisKolom; j++) {
            scanf("%d", &matrixA[i][j]);
        }
    }

    printf("Matriks B\n");
    int matrixB[BarisKolom][BarisKolom];
    for (int i = 0; i < BarisKolom; i++) {
        for (int j = 0; j < BarisKolom; j++) {
            scanf("%d", &matrixB[i][j]);
        }
    }

    int matrixHasil[BarisKolom][BarisKolom];
    for (int i = 0; i < BarisKolom; i++) {
        for (int j = 0; j < BarisKolom; j++) {
            matrixHasil[i][j] = 0;
            for (int k = 0; k < BarisKolom; k++) {
                matrixHasil[i][j] += matrixA[i][k] * matrixB[k][j];
            }
        }
    }

    printf("Matriks AXB\n");
    for (int i = 0; i < BarisKolom; i++) {
        for (int j = 0; j < BarisKolom; j++) {
            printf("%d ", matrixHasil[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: PRAK605_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.py
BarisKolom = int(input())
print('Matriks A')
matrixA = []
for i in range(BarisKolom):
    matrixA.append(list(map(int, input().split())))
print('Matriks B')
matrixB = []
for i in range(BarisKolom):
    matrixB.append(list(map(int, input().split())))
matrixHasil = []
for i in range(BarisKolom):
    matrixHasil.append([0] * BarisKolom)
for i in range(BarisKolom):
    for j in range(BarisKolom):
        for k in range(BarisKolom):
            matrixHasil[i][j] += matrixA[i][k] * matrixB[k][j]
print('Matriks AXB')
for i in matrixHasil:
    print(i)